justfile

```diff
@@ -165,17 +165,21 @@ _api-install:
     "Waiting for the API to be healthy..."
 
 # Run API tests inside Docker
-api-test docker_args="" tests="": up wait-for-es wait-for-ing wait-for-web
-    docker-compose exec {{ docker_args }} web ./test/run_test.sh {{ tests }}
+@api-test docker_args="" tests="":
+    docker-compose exec {{ docker_args }} ./test/run_test.sh {{ tests }}
```

Comment:
Were `web` and the other dependencies intended to be removed here?

Reply:
No

Reply:
I was trying to deduplicate the dependencies list; alas, the `docker_args` argument is a pain in the butt to try to pass through empty.